docs/core-protocol/public-lock/hooks.md

```diff
@@ -9,14 +9,66 @@ sidebar_position: 3
 
 Hooks have been added to strategic places of the Lock contract to allow lock managers to modify behaviour by calling a 3rd party contract.
 
-We currently (version 11) support 5 hooks:
+We currently support 7 hooks (as of v12):
 
+- `onGrantKeyHook`: called when a key is granted
```

Comment:
Let's add anchors to the sections below?

Reply:
Good point. I added anchors too!